website/docs/operator/data-recovery.md

```diff
@@ -4,4 +4,288 @@ title: Data Recovery
+          {
+            "regex": ".*.tar.gz",
+            "sampleFileName": "blah.tar.gz",
+            "bucket": "public"
+          }
+        ]
+      }
+    },
+    "cumulus_meta": {
+      "system_bucket": "test-internal",
+      "asyncOperationId": "1234"
+    }
+  }
+```
+
+The following is the corresponding output that the workflow will return if successful.
+
+```json
+
+{
+  "granules": [
+    {
+      "granuleId": "integrationGranuleId",
+      "version": "integrationGranuleVersion",
+      "files": [
+        {
+          "fileName": "MOD09GQ.A2017025.h21v00.006.2017034065104.hdf",
+          "key": "MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065104.hdf",
+          "bucket": "test-orca-primary"
+        }
+      ],
+      "keys": [
+        {
+          "key": "MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065104.hdf",
+          "destBucket": "test-public"
+        }
+      ],
+      "recoverFiles": [
+        {
+          "success": true,
+          "filename": "MOD09GQ.A2017025.h21v00.006.2017034065104.hdf",
+          "keyPath": "MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065104.hdf",
+          "restoreDestination": "test-public",
+          "s3MultipartChunksizeMb": 200,
+          "statusId": 1,
+          "requestTime": "2022-11-16T17:29:19.008088+00:00",
+          "lastUpdate": "2022-11-16T17:29:19.008088+00:00",
+          "completionTime": "2022-11-16T17:29:19.008088+00:00"
+        }
+      ]
+    }
+  ],
+  "asyncOperationId": "1234"
+}
+
+```
+
+### Recovery workflow configurable parameters
+The following are the parameters needed for recovery workflow:
+
+- buckets- AWS S3 bucket mapping used for Cumulus and ORCA configuration. Contains the following properties:
+  - name (Required)- Name of the S3 bucket.
+  - type (Optional)- the type of bucket - i.e. internal, public, private, protected. 
+
+  It can be set up using the following configuration.
+  ```json
+        "config": {
+          "buckets.$": "$.meta.buckets"
+        }
+    ```
+    Example:
+
+    ```json
+    "buckets": {
+            "protected": {"name": "sndbx-cumulus-protected", "type": "protected"},
+            "internal": {"name": "sndbx-cumulus-internal", "type": "internal"},
+            "private": {"name": "sndbx-cumulus-private", "type": "private"},
+            "public": {"name": "sndbx-cumulus-public", "type": "public"}
+        }
+    ```
+- fileBucketMaps- A list of dictionaries that contains details of the configured storage bucket and file regex. Contains the following properties:
+  - regex (Required)- The regex that matches the file extension type.
+  - bucket (Required))- The name of the key that points to the correct S3 bucket. Examples include public, private, protected, etc.
+  - sampleFileName (Optional)- name of a sample file having extension.
+
+  It can be set up using the following configuration.
+  ```json
+        "config": {
+          "fileBucketMaps.$": "$.meta.collection.files"
+        }
+    ```
```

Comment:
Minor bit of indentation formatting here.

Reply:
Bit of extra indent in some of the `Example:` sections and their surrounding formatting further down as well.

Reply:
tried updating everywhere I could find. Let me know if I am missing anything.